include/NovelRT/Transform.h

```diff
@@ -16,42 +19,79 @@ namespace NovelRT {
     bool _active;
 
   public:
+    /**
+     * Creates a new Transform, with the specified position, rotation and scale.
+     * @param position The position.
+     * @param rotation The rotation.
+     * @param scale The scale.
+     */
     Transform(const Maths::GeoVector2<float>& position, float rotation, const Maths::GeoVector2<float>& scale) noexcept;
+    /**
+     * Creates a new Transform with the default values.
+     */
     Transform() noexcept;
 
+    /**
+     * Gets the Axis Aligned Bounding Box based on the position and scale of this transform.
+     */
     inline Maths::GeoBounds getAABB() const {
       auto scale = fmaxf(_scale.getX(), _scale.getY());
       return Maths::GeoBounds(_position, Maths::GeoVector2(scale, scale), 0);
     }
 
+    /**
+     * Gets the bounds of this transform.
+     */
     inline Maths::GeoBounds getBounds() const {
       return Maths::GeoBounds(_position, _scale, _rotation);
     }
 
+    /**
+     * Gets the constant position of this transform.
+     */
     inline const Maths::GeoVector2<float>& position() const {
       return _position;
     }
 
+    /**
+     * Gets the position of this transform.
+     */
     inline Maths::GeoVector2<float>& position() {
       return _position;
     }
 
+    /**
+     * Gets the constant rotation of this transform.
+     */
     inline const float& rotation() const {
       return _rotation;
     }
 
+    /**
+     * Gets the rotation of this transform.
+     */
     inline float& rotation() {
       return _rotation;
     }
 
+    /**
+     * Gets the constant scale of this transform.
+     */
     inline const Maths::GeoVector2<float>& getScale() const {
       return _scale;
     }
 
+    /**
+     * Gets the scale of this transform.
+     */
     inline Maths::GeoVector2<float>& getScale() {
       return _scale;
     }
 
+    /**
+     * Sets the scale of this transform to the specified value.
```

Comment:
Wow how does this method even still exist? You can remove the docs on this one, this should've been nuked.

Reply:
Going away as well